core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -878,6 +878,26 @@ private static RelDataType arrayReturnType(SqlOperatorBinding opBinding) {
           SqlLibraryOperators::arrayReturnType,
           OperandTypes.SAME_VARIADIC);
 
```

Comment:
comments:
so it should add @SuppressWarnings("argument.type.incompatible") like others
```
[Task :core:compileJava] [argument.type.incompatible] incompatible argument for parameter type of createTypeWithNullability. type = opBinding.getTypeFactory().createTypeWithNullability(type, true);
```

Reply:
The ci failed with this.  So i can see it